### PR TITLE
Nick style

### PR DESCRIPTION
**Pull request by NickMo91**
I have added some basic styling in the home page.

I had some issues with adding icons on some buttons. On the home page on the upper right side, logo button for example.  
header-container-nav-logo {
+	background-image: url("/images/cam.png");
+	float: right;
+	}.
And on the profile page upload button. 
upload-button {
+	background-image: url("upload.png");
+}

### CODE REVIEW

**Comment by wbobeirne** [comment]
Hey @Moyseos, it looks like you added a ton of user-uploaded images with this commit. Can y'all make sure those are gitignored, and don't get uploaded with this branch? You can remove them by doing `git rm -rf path/to/image/folder` and committing that to remove them

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 node_modules/
 *.log
 .env
+assets/images
```
Comment (wbobeirne) [inline .gitignore]: You don't want to gitignore these! These are your static image assets, things like logos / icons etc.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+assets/previews
+uploads
```
Comment (wbobeirne) [inline .gitignore]: You should also ignore assets/thumbnails

File: assets/css/style.css [hunk 3]
```diff
@@ -87,3 +119,89 @@ button {
...
+}
+
+
+.upload-form {
+ margin-right: 25%;
+ margin-left: 25%;
+}
+
+.upload-button {
+	background-image: url("upload.png");
```
Comment (wbobeirne) [inline assets/css/style.css]: This needs to be a full path to `upload.png`, not just the name. How are other images referenced in here?

**Review by wbobeirne — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (NickMo91) [reply .gitignore]: I changed this. I think it should be fine now.

File: .gitignore (hunk 2 above)
Comment (NickMo91) [reply .gitignore]: I did this too.

File: assets/css/style.css (hunk 3 above)
Comment (NickMo91) [reply assets/css/style.css]: This I will revisit soon, cause i want to work on other things.

**Pull request merged**